staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -573,6 +574,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 				handler = restfulGetResource(getter, exporter, reqScope)
 			}
 			handler = metrics.InstrumentRouteFunc(action.Verb, resource, subresource, handler)
+			handler = genericfilters.RestfulWithCompression(handler, a.group.Context)
```

Comment:
where should the escape flag be read from? does this involve adding a configuration variable & a command line flag to apiserver options?